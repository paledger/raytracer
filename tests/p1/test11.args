firsthit ../resources/simple.pov 640 480 400 240
