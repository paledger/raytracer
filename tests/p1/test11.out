Pixel: [400, 240] Ray: {0 0 14} -> {0.1654 0.001027 -0.9862}
No Hit
